CommonTools/RecoUtils/src/precompile.h

```diff
@@ -0,0 +1,13 @@
+// for integration test 
```

Comment:
This produces a 250MB gch file
(TrackExtra is most probably a major contributor)